README.md

```diff
@@ -38,7 +38,7 @@ stable build of Calico following the instructions
 [here](http://www.projectcalico.org/download/).
 
 Technical documentation is
-[here](https://github.com/Metaswitch/calico-docs/wiki);
+[on ReadTheDocs](http://docs.projectcalico.org/);
```

Comment:
I'm not sure we need to advertise that we use RTD, so I think this can still say [here].